pkg/dataprotection/backup/request.go

```diff
@@ -117,7 +123,7 @@ func (r *Request) BuildActions() ([]action.Action, error) {
 	}
 
 	appendIgnoreNil(preBackupActions...)
-	appendIgnoreNil(backupDataAction, createVolumeSnapshotAction, backupKubeResourcesAction)
```

Comment:
These actions should be sequential, starting with the preBackupActions followed by the backupData actions.